docs/docusaurus/docs/core/installation_and_setup/manage_data_contexts.md

```diff
@@ -181,12 +181,9 @@ If you're using GX for multiple projects, you might want to use a different Data
 
 ### Prerequisites
 
-<Prerequisites requirePython = {false} requireInstallation = {true} requireDataContext = {false} requireSourceData = {null} requireDatasource = {false} requireExpectationSuite = {false}>
-
+- A Great Expectations instance.
```

Comment:
I think I should add a link here but I was not sure how to write it correctly in markdown so I just deleted it (it used to be a `VersionedLink` in the prerequisites component). Please @Rachel-Reverie @kwcanuck feel free to tell me to update it as you think it should be